src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
In the case that the there's multiple TFMs selected the outer build will invoke my `buildCrossTargeting` target which will invoke the inner build with the first target framework. All this is orchestrated by how NuGet restores the packages and generates their nuget.targets file. Example:

```XML
<?xml version="1.0" encoding="utf-8" standalone="no"?>
<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
    <NuGetPackageRoot>$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
  </PropertyGroup>
  <ImportGroup Condition=" '$(TargetFramework)' == '' AND '$(ExcludeRestorePackageImports)' != 'true' ">
    <Import Project="$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets" Condition="Exists('$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\buildCrossTargeting\Microsoft.NET.Sdk.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\buildCrossTargeting\Microsoft.NET.Sdk.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\buildCrossTargeting\Microsoft.AspNetCore.Razor.Design.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\buildCrossTargeting\Microsoft.AspNetCore.Razor.Design.targets')" />
  </ImportGroup>
  <ImportGroup Condition=" '$(TargetFramework)' == 'net451' AND '$(ExcludeRestorePackageImports)' != 'true' ">
    <Import Project="$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\build\NuGet.Build.Tasks.Pack.targets" Condition="Exists('$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\build\NuGet.Build.Tasks.Pack.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\build\Microsoft.NET.Sdk.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\build\Microsoft.NET.Sdk.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\build\netstandard1.0\Microsoft.AspNetCore.Razor.Design.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\build\netstandard1.0\Microsoft.AspNetCore.Razor.Design.targets')" />
  </ImportGroup>
  <ImportGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' AND '$(ExcludeRestorePackageImports)' != 'true' ">
    <Import Project="$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\build\NuGet.Build.Tasks.Pack.targets" Condition="Exists('$(NuGetPackageRoot)nuget.build.tasks.pack\3.6.0-rc-1984\build\NuGet.Build.Tasks.Pack.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\build\Microsoft.NET.Sdk.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.net.sdk\1.0.0-alpha-20161019-1\build\Microsoft.NET.Sdk.targets')" />
    <Import Project="$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\build\netstandard1.0\Microsoft.AspNetCore.Razor.Design.targets" Condition="Exists('$(NuGetPackageRoot)microsoft.aspnetcore.razor.design\1.0.0\build\netstandard1.0\Microsoft.AspNetCore.Razor.Design.targets')" />
  </ImportGroup>
</Project>
```